src/styles/blocks/blocks.scss

```
@@ -0,0 +1,30 @@
+.page {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill,
+  #{"minmax(max(300px, (33.3%)), 1fr)"});
+}
+
+.container__box {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
```

You should create a variable for height.
```suggestion
  height: $cell-height;
```

---

Agree, you use 300px limit in other places as well.